internal/pkg/config/config.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+const (
+	// OperatorName is the name when referring to the operator.
+	OperatorName = "seccomp-operator"
+
+	// OperatorNamespace is the default namespace for the operator deployment.
+	OperatorNamespace = OperatorName
```

Comment:
How would you feel about moving theses in `test/`? It appears that is the only place they are used and they could easily get out of sync with the actual deployment if the values were changed there. I would prefer not to the illusion that these values are always correct and instead use the downward API in places where these values are needed within the operator.

Reply:
Happy to follow with this if you prefer :)

Reply:
Yep, I completely removed it and sicked to `OperatorName` in the `test` package.

Reply:
Hmm I think I have the same feeling about `OperatorName` it is not strictly guaranteed to be correct, so it seems safer to be in `test/`